docs/sources/administration/configuration.md

```diff
@@ -1128,11 +1128,11 @@ Sets the alert notification timeout. Default value is `30`.
 
 ### max_attempts
 
-Sets a maximum limit on attempts to sending alert notifications. Default value is `3`.
+Sets a maximum limit on attempts to sending alert notifications. This configuration is also supported in [Grafana 8 Alerts]({{< relref "../alerting/unified-alerting/_index.md" >}}. Default value is `3`.
 
 ### min_interval_seconds
 
-Sets the minimum interval between rule evaluations. Default value is `1`.
+Sets the minimum interval between rule evaluations. This configuration is also supported in Grafana 8 Alerts. For dashboard alerts the default value is `1`, for [Grafana 8 Alerts]({{< relref "../alerting/unified-alerting/_index.md" >}}#min-interval-seconds) the default value is `10` that equals each alerting system's scheduler interval.
```

Comment:
I think we should also specify that it must be a multiple of 10, else the default of 10 will be returned.

Reply:
Plus one, as it was not clear to me.